### DISCUSSION TITLE
How to get rid of the 8 seconds wait after powering a Micronucleous Attiny88?

### DISCUSSION THREAD

**Discussion by Rimbaldo** [SpenceKonde/ATTinyCore#662] [Support / Q & A]
HI! 

Is there an option to disable that 8 second fast blinking wait so the MH_TINY (Micronucleous Attiny88 clone) would execute the code as soon as it's powered on? Or just enter the wait for usb connection when pressing the reset button?

I tried some of the menu options for "Burn Boot entry mode) but none seemed to work...  

Solved: I first updated the bootloader  from the stock one on the board to the AttinyCore and then it worked!

Thanks, regards, Rodrigo

**Comment by SpenceKonde** [comment]
Use an entry mode that doesn't classify POR as an entry condition. This is supported in ATTinyCore 2.0.0, available for public test by manually installing the 2.0.0-dev branch. Please open an issue for any problems you find - testing has been extremely light, and we don't even have CI compile testing here. 
(oh,and it's a 6 second delay according to the source code). 

You will need to update the bootloader either via ISP or the USB bootloader upgrade method. 

The only known issue is that the tiny828 optiboot bootloaders are busted. But I am quite certain there are many serious issues unknown to me.
